packages/docusaurus-utils/src/index.ts

```diff
@@ -197,13 +197,13 @@ export function normalizeUrl(rawUrls: string[]): string {
       // Removing the starting slashes for each component but the first.
       component = component.replace(/^[/]+/, '');
     }
-    if (i < urls.length - 1) {
-      // Removing the ending slashes for each component but the last.
-      component = component.replace(/[/]+$/, '');
-    } else {
-      // For the last component we will combine multiple slashes to a single one.
-      component = component.replace(/[/]+$/, '/');
-    }
+
+    /**
+     * Removing the ending slashes for each component but the last.
+     * For the last component we will combine multiple slashes to a single one.
+     */
+    replaceWith = i < urls.length - 1 ? '' : '/';
```

Comment:
Ditto with the above wrt comments and naming. It's a bad idea to reuse variable. We rather newly another variable or just inline it since this is short enough.